backend-modules/koios/src/main/java/com/bloxbean/cardano/client/backend/koios/KoiosAccountService.java

```diff
@@ -148,6 +148,27 @@ private Result<List<AccountHistory>> convertToAccountHistories(List<AccountHisto
         return Result.success("OK").withValue(accountHistoryList).code(200);
     }
 
+    @Override
+    public Result<List<AccountAddress>> getAllAccountAddresses(String stakeAddress) throws ApiException {
+        List<AccountAddress> accountAddresses = new ArrayList<>();
+        int page = 1;
+        Result<List<AccountAddress>> accountAddressesResult = getAccountAddresses(stakeAddress, 1000, page);
```

Comment:
For BF impl, the page size is 100. But for Koios it's 1000. Any reason why it's not 100?

Reply:
Koios Max Page Size is 1000, for BF is 100